transformation/src/main/scala/org/broadinstitute/monster/encode/transformation/AntibodyTransformations.scala

```diff
@@ -8,19 +8,13 @@ import upack.Msg
 object AntibodyTransformations {
 
   /** Transform a raw ENCODE antibody into our preferred schema. */
-  def transformAntibody(antibodyInput: Msg): Antibody = {
+  def transformAntibody(antibodyInput: Msg, joinedTargets: Iterable[Msg]): Antibody = {
     import org.broadinstitute.monster.common.msg.MsgOps
 
-    // Use regular expressions to remove everything but the actual target name. Remove any non-human targets.
-    val mappedTargets = antibodyInput
-      .tryRead[Array[String]]("targets")
-      .getOrElse(Array.empty)
-      .map(CommonTransformations.transformId)
-      .flatMap { target =>
-        val (front, back) = target.splitAt(target.lastIndexOf('-'))
-        if (back == "-human") Some(front)
-        else None
-      }
+    val targetNames = joinedTargets
+      .filter(_.read[String]("organism") == "/organisms/human/")
+      .map(_.read[String]("label"))
+      .toArray
```

Comment:
This part LGTM!

Reply:
Oh wait, sorry. This needs a `.sorted.distinct` to ensure reproducibility, since there's no guarantee on the order of `joinedTargets`